Dangerfile.df.kts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2021. D4L data4life gGmbH / All rights reserved.
+ *
+ * D4L owns all legal rights, title and interest in and to the Software Development Kit ("SDK"),
+ * including any intellectual property rights that subsist in the SDK.
+ *
+ * The SDK and its documentation may be accessed and used for viewing/review purposes only.
+ * Any usage of the SDK for other purposes, including usage for the development of
+ * applications/third-party applications shall require the conclusion of a license agreement
+ * between you and D4L.
+ *
+ * If you are interested in licensing the SDK for your own applications/third-party
+ * applications and/or if you’d like to contribute to the development of the SDK, please
+ * contact D4L by email to help@data4life.care.
+ */
+
+import systems.danger.kotlin.danger
+import systems.danger.kotlin.fail
+import systems.danger.kotlin.onGitHub
+import systems.danger.kotlin.warn
+
+danger(args) {
+    val allSourceFiles = git.modifiedFiles + git.createdFiles
+    val isChangelogUpdated = allSourceFiles.contains("CHANGELOG.adoc")
+
+    onGitHub {
+        val branchName = pullRequest.head.label.substringAfter(":")
+        val isFeatureBranch =
+            "(?:feature\\/(?:[A-Z]{2,8}-\\d{1,6}\\/)?(?:add|change|remove|fix|bump|security)-[a-z,-]*)"
```

Comment:
`[a-z,-]` don't we allow also number?

Reply:
that's the check for the ticket, e.g `feature/SDK-456/add-some-awesome-stuff`

Reply:
Indeed, but it would not allow `feature/SDK-456/change-some-base64-stuff`, right? That's more what I aiming for and if we want that or not?

Reply:
it allows `add|change|remove|fix|bump|security`

Reply:
ahh, now I got it, no numbers allowed, could be changed

Reply:
done